lib/cc/cli/runner.rb

```diff
@@ -8,8 +8,8 @@ def self.run(argv)
         new(argv).run
       rescue => ex
         $stderr.puts("error: (#{ex.class}) #{ex.message}")
-
         CLI.debug("backtrace: #{ex.backtrace.join("\n\t")}")
+        exit 1
```

Comment:
EDIT: looks like it's in its own commit, so I'm fine with it (unless it causes enough issues to be worth pulling for now)